### PR TITLE
implement behavior after choosing a date and time from top selections

### PR DESCRIPTION
**Pull request by sapirshp**

### CODE REVIEW

File: app/src/main/res/layout/date_setup_popup.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginTop="10dp"
+                android:background="@color/backgroundColor"
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/textView4"
```
Comment (Chen-Ben-yashar) [inline app/src/main/res/layout/date_setup_popup.xml]: change to inicadite name

File: app/src/main/java/com/example/meetapp/MenuHandler.java [hunk 2]
```diff
@@ -281,16 +290,20 @@ void handleCreateMeeting(final CalendarSlotsHandler calendarSlotsHandler)
...
-        TextView exitBtn = topSuggestionsDialog.findViewById(R.id.exitPopupBtn);
-        handleExitPopup(topSuggestionsDialog, exitBtn);
-        Button[] allOptionsLst = createAllOptionsLst();
-        final ArrayList<Button> currentOptionLst = activateOnlyRelevantButtons(allOptionsLst,
-                numOfOptionsToDisplay);
-        setTextForOptions(currentOptionLst, calendarSlotsHandler);
-        handleAllOptionPresses(currentOptionLst, allOptionsLst);
-        handleCreateMeetingBtnPress(currentOptionLst.get(currentOptionLst.size()-1));
-        topSuggestionsDialog.show();
+        if (!isDateChosen) {
```
Comment (Chen-Ben-yashar) [inline app/src/main/java/com/example/meetapp/MenuHandler.java]: wrap the inside of the if statement in a separate function.
it is too long

File: app/src/main/res/layout/date_setup_popup.xml (hunk 1 above)
Comment (sapirshp) [reply app/src/main/res/layout/date_setup_popup.xml]: fixed

File: app/src/main/java/com/example/meetapp/MenuHandler.java (hunk 2 above)
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/MenuHandler.java]: fixed

**Pull request merged**